db/orders.js

```diff
@@ -48,7 +46,7 @@ async function getOrderItemByUser({ id }) {
     }
 }
 
-async function updateOrderItem({ id, ...fields }) {
+async function updateOrder({ id, ...fields }) {
```

Comment:
I think we only ever want to update the status of an order since userId and date shouldn't be changed so instead of ..fields it could just require a status parameter and we could call it updateOrderStatus. This way we avoid the possibility of unwanted changes from bad or malicious api calls.